Projects/Features/Scene/GroupDetailScene/Vote/VoteComplete/VoteCompleteView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  VoteCompleteView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 7/6/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import DesignSystem
+import SwiftUI
+
+struct VoteCompleteView: View {
+  private let store: StoreOf<VoteCompleteCore>
+
+  init(store: StoreOf<VoteCompleteCore>) {
+    self.store = store
+  }
+
+  var body: some View {
+    VStack {
+      Spacer()
+      
+      Text("내 PIC을 골랐어요!")
+        .font(.head20)
+        .foregroundStyle(DesignSystem.Colors.gray80)
+        .padding(.bottom, 16)
+      
+      Text("모든 사람이 PIC을 완료하면\n네컷 사진이 만들어져요")
+        .font(.body14)
+        .foregroundStyle(DesignSystem.Colors.gray60)
+        .multilineTextAlignment(.center)
+        .padding(.bottom, 24)
+      
+      // TODO: 랜덤 이미지 프레임 씌운 view로 변경
+      RoundedRectangle(cornerRadius: 10)
+        .foregroundStyle(.red)
+        .frame(width: 240, height: 240)
+        .padding(.horizontal, 76)
+      
+      Spacer()
+      Spacer()
```

Comment:
P4:
한 줄 띄어써도 좋을 것 같습니다
```suggestion
      Spacer()
      
      Spacer()
```